.nvmrc

```diff
@@ -0,0 +1 @@
+16.3.0
```

Comment:
I would suggest to use `12.10.0` or `12.22.7`, since this library is used on `cloud-frontend` repo that needs to run on Node `v12.10.0`, so all the `dependencies` need to be compatible with that version... and as for `12.22.7`, since most of the GitHub checks on our repos run on node v12, the last compatible version with these is Node `v12.22.7`, so all the `devDependencies` and `peerDependencies` need to be compatible with this version.
For the last part, I am not 100% sure if this applies to `charts` as well! 😄